app/views/facilities/_facilities_list.html.haml

```diff
@@ -1,9 +1,9 @@
 - if @facilities.any?
-  %h4.facility_listing.js--facility_listing.all_header= text(".all")
+  %h2.facility_listing.js--facility_listing.all_header= text(".all")
   - @azlist.map.with_index do |(letter, matched_facilities), index|
     - matched_facilities.each do |facility|
       .facility_listing.js--facility_listing{ class: az_classname_for_facility(index, letter) }
-        %h4.header-tight= link_to facility, facility_path(facility)
+        %h3.header-tight= link_to facility, facility_path(facility)
```

Comment:
I'm not sure if it's just because it's different from what it was before, but the size feels a bit too much if this is an `h3`. Perhaps a header is not the right element for this element.